wgsl/index.bs

```diff
@@ -6973,6 +6999,15 @@ textureSample(t : texture_depth_cube_array, s : sampler, coords : vec3<f32>, arr
 
 The sampled value.
 
+### `textureSampleLuminance` ### {#texturesampleluminance}
+
+Samples a luminance value from a `texture_external`.
```

Comment:
What is the range of the resulting values?  0.0 to 1.0?

Reply:
Good question. I think in the case of an HDR source we would want this to exceed 1.0, but 1.0 would be the max SDR luminance. The same would actually be true of RGB output (which could exceed 1.0 or even be negative, I believe).